Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.13 AS build-env
+WORKDIR /go/src/github.com/fairwindsops/pluto/
+
+ENV GO111MODULE=on
+ENV GOPROXY=https://proxy.golang.org
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN go build -a -o pluto *.go
+
+FROM alpine:3.10
```

Comment:
Please update to a more current version

Reply:
Also, is there a good reason to not use scratch like we do in rbac-manager or goldilocks?

https://github.com/FairwindsOps/goldilocks/blob/master/Dockerfile

Reply:
Went scratch - good call